README.md

```diff
@@ -56,3 +56,8 @@ data from `localhost:8000` **where the Django app must be running**.
 
 ### Start dev server
 - `npm start`
+
+##CLI
+1. Run pipenv shell.
+2. Create a workflow using UI and save it. 
+3. Run it as: pyworkflow --file-directory (path-to-json-workflow-file) execute
```

Comment:
I kept getting a `command not found` error when trying to the CLI as `pyworkflow`. Following the [Click guide](https://click.palletsprojects.com/en/7.x/setuptools/#testing-the-script), adding 

`pip install -editable .`

made it work. Not sure how/if this is compatible with `pipenv` (I ran the command within the `pipenv shell`), but it let me test it!

Reply:
Good call on installing that package. Will add that in the README file.